app/services/katello/event_monitor/poller_thread.rb

```diff
@@ -0,0 +1,85 @@
+module Katello
+  module EventMonitor
+    class PollerThread
+      SLEEP_INTERVAL = 2
```

Comment:
Where did the 2 second value come from? https://devblogs.microsoft.com/oldnewthing/20190924-00/?p=102915

Reply:
hah, i think it was just the idea of we don't want it to wait too long (30 seconds might be too long).  It can be changed to 1 or 3 most likely

Reply:
CandlepinListeningService had a similar sleep, but was just 1 second

Reply:
I think I like 3 too